services/FeefoReviewService.php

```diff
@@ -1,26 +1,24 @@
 <?php
 namespace Craft;
 
-class FeefoReviewService extends BaseApplicationComponent
-{
+class FeefoReviewService extends BaseApplicationComponent {
   public $plugin = null;
   public $settings = array();
 
-  public function __construct()
-  {
+  public function __construct(){
```

Comment:
There was inconsistencies in how we placed the braces in the previous code but you've changed it to be inconsistent in another way.

Let's settle on having the opening brace on the same line as the definition with a space after the name/arguments like so:
```
public function __construct() {
  ...
}
```
Please update all definitions in this file to match.